qa/rpc-tests/getrawtransaction_insight.py

```diff
@@ -61,7 +58,6 @@ def run_test(self):
         tx_b = self.nodes[2].getrawtransaction(txid_b, 1)
         assert('height' not in tx_b)
 
-        self.sync_all()
```

Comment:
I think this is not correct because the following `getrawtransaction` may depend on being synced.

Reply:
```suggestion
        self.sync_all()
```